README.md

```diff
@@ -45,9 +45,16 @@ To delete all key-value pairs in the Tags field of a specific HANA instance:
 ```
 az hanainstance update --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME --set tags={}
 ```
+
+To enable monitoring for a specific HANA instance:
+
+```
+az hanainstance enable-monitoring --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME --hana-vnet $HANA_VNET  --hana-hostname $HANA_HOSTNAME --hana-instance-num $HANA_INSTANCENUM --hana-db-username $HANA_DB_USERNAME --hana-db-password $HANA_DB_PW
```

Comment:
There is an extra space between `$HANA_VNET` and `--hana-hostname`

I suggest to split this line to improve readability.